### DISCUSSION TITLE
AWS Lightsail and jamulus-headless 3.7.0 install issues

### DISCUSSION THREAD

**Discussion by BazzaYoung** [jamulussoftware/jamulus#1355] [Technical Support & Help] [answered]
Firstly, thanks to all contributors for this great project.  I'm trying to get a private server up and running for our band rehearsals and am following Simon's Idiots Guide (as I'm pretty basic on Linux although IT experienced). 

However I ran into fundamental issues on Ubuntu 18.04 with dpkg package dependency errors. With Ubuntu 20.04 now available I also tried the installation on this version as well but encountered the same issues. In both cases the following packages do not appear to be installed by default on these Lightsail instances: -

- libqtcore5a
- libqt5network5
- libqt5xml5

Can I resolve this by simply(?), running the following command for each package "sudo apt-get install xxxxxx" or am I just digging a hole for myself?

Additionally is Ubunto 20.04 supported or do I need to stick to 18.04?

Many thanks in advance
Barry

**Comment by gilgongo** [comment]
I think for some reason for a fresh Lightsail install you need to run `sudo apt update` before anything else otherwise it won't find some packages (in fact we do say that in the install [instructions](https://jamulus.io/wiki/Server-Linux) now but they've only recently been updated).

20.04 should be fine.

**Reply by BazzaYoung** [reply]
Thanks for such a prompt reply, very much appreciated.  Apologies if I have missed the latest instructions, I was pulling them from a faceache link.

I'll crack on and give this a try

**Comment by hoffie** [comment, marked as answer]
I'm wondering whether the script uses `dpkg -i` instead of a command which resolves dependencies automatically (`apt install`)?

**Reply by gilgongo** [reply]
To which script are you referring?

**Reply by BazzaYoung** [reply]
If it helps... I'm pulling the script "https://worldjam.vip/jamulus/setup-jamulus.sh" which has uses "dpkg -iEG"

**Reply by gilgongo** [reply]
Oh Ok. Probably better to use `apt install` not `dpkg -i` as I think the latter may be too "literal". But `apt update` should do you fine either way.

**Reply by BazzaYoung** [reply]
Many thanks for your kind assistance, all good and up & running.  My linux knowledge is very rusty so it's actually quite useful to dust it off.
